modules/claims_api/config/schemas/2122.json

```diff
@@ -3,17 +3,160 @@
   "description": "Form 2122 Schema",
   "type": "object",
   "required": [
-    "poaCode"
+    "serviceOrganization"
   ],
-  "properties": {
-    "poaCode": {
-      "type": "string"
+  "definitions": {
+    "addressLine": {
+      "type": "string",
+      "pattern": "^([-a-zA-Z0-9'.,&#]([-a-zA-Z0-9'.,&# ])?)+$",
+      "maxLength": 20
+    },
+    "date": {
+      "type": "string",
+      "pattern": "^(\\d{4}|XXXX)-(0[1-9]|1[0-2]|XX)-(0[1-9]|[1-2][0-9]|3[0-1]|XX)$"
```

Comment:
this regex allows stuff like:
`1985-XX-31`
`XXXX-08-XX`
do we want that?
what are the `XXXX-XX-XX` dates for?